### PR TITLE
Introduce SAI and Managed Services

### PR DESCRIPTION
**Pull request by trentsteenholdt-purple**
With the Career Framework endorsed by SAI and MS, introducing the new portfolios to this site.

### CODE REVIEW

File: src/routes/[slug].svelte [hunk 1]
```diff
@@ -36,12 +36,14 @@
 	<title>Purple Personas - {post.metadata.title}</title>
 </svelte:head>
 
-<div class="px-12 lg:px-32 max-w-screen-xl">
-	<h2 class="pb-4 lg:pb-10 text-black text-xl">Our roles</h2>
-	<div class="flex flex-col lg:flex-row justify-between pb-4 lg:pb-14">
+<div class="px-12 lg:px-32 max-w-screen bg-gray">
```
Comment (FizzBuzz791) [inline src/routes/[slug].svelte:39]: I don't think there's a `max-w-screen` option, looking at the [docs](https://tailwindcss.com/docs/max-width#constraining-to-your-breakpoints)?

**Review by FizzBuzz791 — APPROVED** [review]
LYW 👊

File: src/routes/[slug].svelte (hunk 1 above)
Comment (trentsteenholdt-purple) [reply src/routes/[slug].svelte]: Yep, good catch. Have fixed :)

**Review by FizzBuzz791 — APPROVED** [review]
LGTM 🚀

**Pull request merged**